docs/maya/tools.md

```diff
@@ -77,7 +77,13 @@ In the specific case of muscle deformers, the too will display the following att
     - Masses are set to 1.0 by default. This will mean that by default the solver will consider that the muscle has a uniform mass.
 <!-- Note for later to update this information once the Global Damping attribute is added-->
 <!-- Note for later to update this information once the Slide On Geometry Damping attribute is added-->
-  - **Slide on Segment Constraints**
+  - **Slide On Geomety**
+    - If this attribute is selected, a list widget is shown with the names of the targets connected to the deformer (Figure 8).
+    - Select the desired target to paint from the list widget and paint the weight values.
+    - When selecting a target in the list, the object will also get selected in the scene, facilitating its identification.
+    - If more than one target was added to the system, then the paint tool will normalize the weights automatically after a stroke has been completed, meaning that the sum of all slide on geometry constraint weights in a vertex will always add up to a maximum value of 1.0.
+    - If any target is removed or added to the system, then the paint tool will refresh the list on mouse hover over the UI.
+  - **Slide On Segment**
```

Comment:
Was there a reason why we no longer call it "Slide On Segment Constraint"

Reply:
I've removed "constraint" because of how the names are exposed in the Paint Tool UI. Let me know if you agree.
![image](https://github.com/Inbibo/adonisfx_docs/assets/150016193/006f18de-9c13-4172-a5f5-554a28383754)